### ISSUE TITLE
Mesos Plugin Configuration for version 2.0.beta -20

### ISSUE THREAD

**Issue by MHarris021** [jenkinsci/mesos-plugin#405]
Does version 2.x of the plugin no longer require the Mesos libraries to be installed on the Jenkins Server? I ask, because the option to configure their location no longer appears in the UI. If they are required, do they need to be installed in a specific location?

**Comment by kaiwalyajoshi** [comment]
@MHarris021 yes, version 2.x uses [Mesos v1 HTTP API](http://mesos.apache.org/documentation/latest/scheduler-http-api/) via [USI](https://github.com/mesosphere/usi) and no longer requires the Mesos libraries.
